python/sglang/srt/layers/moe/ep_moe/layer.py

```diff
@@ -47,12 +47,17 @@
     get_bool_env_var,
     is_hip,
     is_npu,
+    next_power_of_2,
 )
 
 _is_hip = is_hip()
 _is_npu = is_npu()
 _is_fp8_fnuz = is_fp8_fnuz()
 _use_aiter = get_bool_env_var("SGLANG_USE_AITER") and _is_hip
+use_flashinfer_blockscale_fp8_moe = (
+    global_server_args_dict["enable_flashinfer_trtllm_moe"]
+    and global_server_args_dict["enable_ep_moe"]
+)
```

Comment:
nit:

* shall we name it a bit more consistent - e.g. use_flashinfer_trtllm_moe
* and in serverargs, assert enable_ep_moe is enabled when we have enable_flashinfer_trtllm_moe

Reply:
Done. PTAL.